week-4-practise/level-1/exercise-1.js

```diff
@@ -38,7 +38,34 @@ var writers = [
     alive: true
   }
 ];
-
+writers.forEach(function(param) {
+  console.log(
+    "Hi, my name is " +
+      param.firstName +
+      param.lastName +
+      ". " +
+      "I am " +
+      param.age +
+      " years old, and work as a " +
+      param.occupation
+  );
+});
 /*
 If you want an extra challenge, only `console.log()` the writers that are alive.
 */
+function aliveWriters() {
+  return writers
+    .filter(name => name.alive == true)

```

Comment:
I would just use `name.alive` without the `==`. If you are comparing values, you should always use the `===`. I don't think it matters here, but it's a good habit in Javascript.

The parameter `name` isn't a name, it points to a "writer" object. So it would be better called `writer`.